### PR TITLE
Issue #13213: Removed //ok from nowhitespaceafter, singlespaceseparator, whitespaceafter

### PR DESCRIPTION
**Pull request by mdxabu**
I removed `//ok` from part of issue #13213 ,
- InputNoWhitespaceAfterTestAllTokens.java
- InputNoWhitespaceAfterTestAllowLineBreaks.java
- InputSingleSpaceSeparatorCommentsWithEmoji.java
- InputWhitespaceAfterCountUnicodeCorrectly.java
- InputWhitespaceAfterTypeCast.java

### CODE REVIEW

**Comment by mahfouz72** [comment]
edit the comment message to adhere with checkstyle standands
it has to be like this `Issue #13213: Remove //ok from ...`
also you have to mention the issue in your PR description e.g. `part of issue #13213 ` put this as the first line in your PR description

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -2160,42 +2160,6 @@
...
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:2198]: @mdxabu  ,
It is recommended to try picking a specific module and working on cleaning it up instead. 
I don't mean this is wrong, but it is surely confusing to review :)

**Review by MANISH-K-07 — COMMENTED** [review]
Your commit message needs to adhere to checkstyle rules : 

```
[ERROR] Failures: 
[ERROR]   CommitValidationTest.testCommitMessageHasProperStructure:216 Commit a4bb9d93228dc9f7ebc25fb8d545e7a301bdb014 message: "fix: remove ok comments from InputEqualsHashCode" is invalid
Proper commit message should adhere to the following rules:
    1) Must match one of the following patterns:
        ^Issue #\d+: .*$
        ^Pull #\d+: .*$
        ^(minor|config|infra|doc|spelling|dependency|supplemental): .*$
    2) It contains only one line of text
    3) Must not end with a period, space, or tab
    4) Commit message should be less than or equal to 200 characters

The rule broken was: 1

```

Please change accordingly.
Look at merged PRs for example : 

- #14259 
- #14260 
- #14264

File: config/checkstyle-input-suppressions.xml (hunk 1 above)
Comment (mdxabu) [reply config/checkstyle-input-suppressions.xml:2198]: Ok, I'll work on correct way on upcoming contributions

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterTestAllowLineBreaks.java [hunk 2]
```diff
@@ -286,24 +286,24 @@ public void parentheses() {
 
     public static void testNoWhitespaceBeforeEllipses(String ... args) {
     }
-    @ interface BAD{} // ok
-    @   interface BAD2{} // ok
+    @ interface BAD{}
+    @   interface BAD2{}
     @ // ok
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterTestAllowLineBreaks.java:291]: Please remove the '// ok' comment here.

**Review by MANISH-K-07 — COMMENTED** [review]
Un-suppressed '// ok' comments throw errors as you can see in the failing CI.

```
[INFO] [checkstyle] Running Checkstyle  on 2284 files
[ERROR] [checkstyle] [ERROR] /home/circleci/project/src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterTestAllowLineBreaks.java:291: // ok comments should have an explanation, or they are unnecessary. [UnnecessaryOkComment]
```

**Pull request closed without merging**